pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/AbstractApexRule.java

```diff
@@ -101,13 +101,16 @@
 
 public abstract class AbstractApexRule extends AbstractRule
         implements ApexParserVisitor, ImmutableLanguage, CodeClimateRule {
+	
+	private static final IntegerProperty CODACY_TIME_TO_FIX = new IntegerProperty("codacy_minutes_to_fix", 
+			"Codacy.com Time to fix in minutes", Integer.MIN_VALUE, Integer.MAX_VALUE, 5, 1.0f);
```

Comment:
The constant name does not match the property name.  What does negative time to fix mean?  Does it work without a default value, 5 minutes seems arbitrary?